api/src/database/migrations/20210409A-add-mapbox-api-key.ts

```diff
@@ -2,12 +2,12 @@ import { Knex } from 'knex';
 
 export async function up(knex: Knex) {
 	await knex.schema.alterTable('directus_settings', (table) => {
-		table.string('mapbox_key');
+		table.jsonb('basemaps');
```

Comment:
Lets use regular `json` here for consistency (we use regular JSON everywhere else)